CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/hashmod/HashmodCSVImportFileParser.java

```diff
@@ -51,9 +51,10 @@ public List<String[]> parse(final HashmodInputSource input) throws IOException {
 
                 results.add(line);
             }
-        } catch (final Exception ex) {
-            NotifyDisplayer.display(String.format("Error detected for file %s",
-                    input.getFile().getPath()), NotifyDescriptor.ERROR_MESSAGE);
+        } catch (final NullPointerException | IOException ex) {
```

Comment:
My initial comments were going to be to do a null check rather than catching a NullPointerException. Upon closer inspection though, it seems the catch block isn't necessary as none of the code in the try throws those exceptions it seems.

Reply:
The NPE or IOException may be generated when non-existent file or a directory path (without a filename) is entered in the textarea and the "chained" button is clicked. It seems to come from either the FileInputStream or CSVParser.

Reply:
Ok. Taking another look I can see `getCSVParser` throws an IO Exception. Can we swap the null pointer catch then for a null check (on whatever would cause the null pointer)?  Generally its better practice to do a null check rather than handle the null pointer after the fact